viewer/FractalShading.js

```diff
@@ -64,18 +106,144 @@ const COLOR_BY_DISTANCE = new Cesium.CustomShader({
     fragmentShaderText: `
     void fragmentMain(FragmentInput fsInput, inout czm_modelMaterial material) {
         float dist_from_center = length(fsInput.attributes.positionMC);
-        float freq = 8.0;
+        float freq = 4.0;
         float wave = 0.5 + 0.5 * cos(2.0 * czm_pi * freq * dist_from_center);
         material.diffuse *= wave;
     }
     `
 });
 
+const COLOR_OCTANTS = new Cesium.CustomShader({
+    lightingModel: Cesium.LightingModel.UNLIT,
+    vertexShaderText: `
+    void vertexMain(VertexInput vsInput, inout czm_modelVertexOutput vsOutput) {
+        vsOutput.pointSize = 4.0;
+    }
+    `,
+    fragmentShaderText: `
+    void fragmentMain(FragmentInput fsInput, inout czm_modelMaterial material) {
+        vec3 position = fsInput.attributes.positionMC;
+        vec3 octants = step(0.0, position);
+        material.diffuse = octants;
+    }
+    `
+});
+
+
+const VIEW_CLUSTER_COORDINATES = new Cesium.CustomShader({
```

Comment:
Add More shaders for procedurally coloring the cluster coordinates. These can be specialized for the particular cluster shapes if necessary (e.g. make one that looks good for circles, another one that looks good for disks, etc.